lib/routes/account.js

```diff
@@ -390,6 +395,26 @@ module.exports = function (
             )
         }
 
+        function checkNumberOfActiveSessions () {
+          return db.sessions(emailRecord.uid)
```

Comment:
Is it worth splitting into separate `getActiveSessions` and `checkNumberOfActiveSessions` functions to make this clearer?